src/common/PageListFilters.tsx

```diff
@@ -30,9 +37,13 @@ export default function PageListFilters({ tabs, onSearchInput, actionBar, ...res
               <UiIcon componentName='search' />
             </InputAdornment>
           ),
+          // TODO: change color
+          sx: { borderRadius: 25, borderColor: theme => theme.palette.grey[300] },
         }}
         placeholder={t('page-list-filters.search-input-placeholder')}
         onChange={e => handleSearchInput(e.target.value)}
+        size={'small'}
+        sx={{ ...inputSx }}
```

Comment:
is it used?

if we have to keep it we can get rid of spread here, just use `sx={inputSx}`